lib/chef/application/solo.rb

```diff
@@ -326,9 +333,9 @@ def fetch_recipe_tarball(url, path)
   end
 
   def unforked_interval_error_message
-    "Unforked chef-client interval runs are disabled in Chef 12." +
+    "Unforked chef-solo interval runs are disabled in Chef 12." +
```

Comment:
Update the spec
https://github.com/chef/chef/blob/c1a389c2a8452e9b796aa1d34c4d9e51f4af30c7/spec/unit/application/solo_spec.rb#L70